src/ngrest/render/RenderCrud.php

```diff
@@ -41,13 +61,23 @@ class RenderCrud extends Render implements ViewContextInterface, RenderCrudInter
      */
     public function getView()
     {
-        if ($this->_view === null) {
-            $this->_view = new RenderCrudView();
-        }
-
         return $this->_view;
     }
     
+    /**
+     * Sets the view object
+     *
+     * @since 2.0
+     */
+    public function setView($value)
+    {
```

Comment:
I assume if `setView()` will not be called, getView will return an empty object, which will then throw an exception while accessing. Therefore i would keep the getter and setter concept:

getView()
+ if null, create the object

```
if ($this->_view === null) {	
    $this->_view = new RenderCrudView();	
}
```

setView($view)
+ allow to set $_view like you did.